src/TestSuite/ConsoleIntegrationTestCase.php

```diff
@@ -13,307 +13,14 @@
  */
 namespace Cake\TestSuite;
 
-use Cake\Console\CommandRunner;
-use Cake\Console\ConsoleInput;
-use Cake\Console\ConsoleIo;
-use Cake\Core\Configure;
-use Cake\TestSuite\Stub\ConsoleOutput;
-
 /**
  * A test case class intended to make integration tests of cake console commands
  * easier.
+ *
+ * @deprecated use Cake\TestSuite\ConsoleIntegrationTestTrait instead
```

Comment:
I guess this makes it easier for application developers to have their own test case hierarchy.

Reply:
Yes. Personally I have a `TestCase` base class that extended IntegrationTestCase so I had that functionality when I needed it. However, when I want a ConsoleIntegrationTest, I wasn't able to use my base class.